components/custom_status/custom_status_modal.tsx

```diff
@@ -129,7 +129,7 @@ const CustomStatusModal: React.FC<Props> = (props: Props) => {
     };
 
     const recentStatuses = (
-        <div>
+        <div className='statusSuggestion__recents'>
```

Comment:
These classes were originally added to easily test that the custom statuses are moved from suggestions to recent statuses on satisfying appropriate conditions (i.e. not related to a different issue). 

This is now changed to use `id` instead of `className`.